internal/monitors/cadvisor/kubeletstats.go

```diff
@@ -153,3 +154,18 @@ func (kip *kubeletInfoProvider) getAllContainersLatestStats() ([]info.ContainerI
 	}
 	return result, nil
 }
+
+func (kip *kubeletInfoProvider) GetEphemeralStatsFromPods() ([]stats.PodStats, error) {
+	req, err := kip.client.NewRequest("POST", "/stats/summary/", nil)
```

Comment:
Add these stats to a group and then conditionalize the request based on whether any metrics in that group are enabled.  There is a helper method on the Output object to do that.

Reply:
Added metrics to a group called `podEphemeralStats`